astropy/cosmology/tests/test_utils.py

```diff
@@ -3,10 +3,7 @@
 import numpy as np
 import pytest
 
-from astropy.cosmology.utils import (
-    aszarr,
-    vectorize_redshift_method,
-)
+from astropy.cosmology.utils import aszarr, vectorize_redshift_method
```

Comment:
I prefer the updated code, but this change is unrelated to what the pull request claims to be doing.

Reply:
I deleted 1 comma and let black do it's thing.

Reply:
I wonder if ruff has a tool to tell if end commas are superfluous...

Reply:
Yes, rule [COM819](https://beta.ruff.rs/docs/rules/prohibited-trailing-comma/). But it would interfere with [Black's magic trailing comma](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma) so I am in not in favor of enforcing that rule.